packages/components/src/utils/dispatchDatadogAction.ts

```diff
@@ -0,0 +1,17 @@
+export const dispatchDatadogAction = (
+  action: string,
+  data: {
+    [key: string]: unknown;
+  },
+  ref?: HTMLElement
```

Comment:
This should probably be `{ dispatchEvent(CustomEvent): void }` and not be optional.